.github/workflows/image-push-release.yml

```diff
@@ -0,0 +1,66 @@
+name: "Image push release"
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  image-build-push:
+    name: Image build and push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set repository as lower-case output variable
+        id: repo_name
+        run: echo ::set-output name=repository::$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker operator image meta
+        id: docker_meta_operator
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ steps.repo_name.outputs.repository }}
+          tags: |
+            type=ref,event=tag
+          flavor: |
+            latest=false
+      - name: Build and push operator image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
```

Comment:
`context` and `file` have sane [defaults](https://github.com/docker/build-push-action?tab=readme-ov-file#customizing), no need to explicitly set them here unless there are high chances of changes to them in near future.